src/elements/content-picker/shareAccessCellRenderer.js

```diff
@@ -15,11 +16,16 @@ export default (
     extensionsWhitelist: string[],
     hasHitSelectionLimit: boolean,
 ) => ({ rowData }: { rowData: BoxItem }) => {
-    if (!isRowSelectable(selectableType, extensionsWhitelist, hasHitSelectionLimit, rowData)) {
+    if (!isRowSelectable(selectableType, extensionsWhitelist, hasHitSelectionLimit, rowData) || !rowData.selected) {
         return <span />;
     }
 
-    return (
+    const { allowed_shared_link_access_levels } = rowData;
+    const isLoading = !allowed_shared_link_access_levels;
+
+    return isLoading ? (
+        <LoadingIndicator className="bcp-ShareAccessSelect--loading" />
```

Comment:
Modifiers should be used together with their base class, like `className="bcp-ShareAccessSelect bcp-ShareAccessSelect--loading"`. Also, since this isn't actually an instance of the ShareAccessSelect component, the class should probably be renamed.

Reply:
Makes sense, how's `bcp-share-access-loading`?